queue/queue.py

```diff
@@ -10,30 +10,35 @@ def __init__(self):
         """
         Initialize the Queue with an empty LinkedList
         """
-        pass
+        super().__init__()
```

Comment:
Since this is not a subclass of a `LinkedList` this line isn't needed.

Reply:
I thought it was a child o, that's why i took so long 😭